client/me/help/help-contact-form/index.jsx

```diff
@@ -109,6 +112,17 @@ export class HelpContactForm extends React.PureComponent {
 		qanda: [],
 	};
 
+	componentWillMount() {
+		const { npsSurveyFeedback, translate } = this.props;
+
+		if ( npsSurveyFeedback ) {
+			this.state.message =
+				'\n' +
+				translate( 'The below comment is copied from your survey response:' ) +
+				`\n${ repeat( '-', 20 ) }\n${ npsSurveyFeedback }`;
```

Comment:
Could we consider to just put 20 `-` characters here?
Importing `repeat` increases the gzipped bundle size by 166 bytes. It's not much but I personally feel this case doesn't quite worth the cost :)

Reply:
Oh, I forgot to mention: I knew that because I had created a test PR so iscalypsofastyet.com would compute for me: https://github.com/Automattic/wp-calypso/pull/34618 , so 626 - 460 = 166.

Reply:
@southp It makes sense. Done.